lms/djangoapps/ccx/tests/test_utils.py

```diff
@@ -185,6 +187,7 @@ def test_auto_enroll_passed_correctly(self):
 class TestEnrollEmail(ModuleStoreTestCase):
     """tests for the enroll_email function from ccx.utils
     """
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
```

Comment:
In fact, @cpennington, the utils method under test *does* construct a CCXLocator in order to put the correct link into the email sent to enrolled students.  So this must be a split modulestore in order to prevent attempting to build a CCXLocator with deprecated Course IDs